src/nile/core/declare.py

```diff
@@ -26,13 +26,13 @@ async def declare(
         file = f"{network}.{DECLARATIONS_FILENAME}"
         raise Exception(f"Alias {alias} already exists in {file}")
 
-    max_fee = "0" if max_fee is None else str(max_fee)
+    max_fee = "0" if max_fee is None else int(max_fee)
```

Comment:
```suggestion
    max_fee = 0 if max_fee is None else int(max_fee)
```
_types are coming_ 👻